### PR TITLE
Maintain one chart for App Mesh controller

### PR DESCRIPTION
**Pull request by fawadkhaliq**
*Description:*
We have been using manager and controller interchangeably for the App Mesh controller. This causes confusion in the documents. Instead of keeping two separate helm charts, we will 
 maintain only one `appmesh-controller` chart moving forward and it will carry the new releases. 

*Changes include:*
- Rename manager to controller in appmesh-manager chart
- Replace appmesh-controller with appmesh-manager (now appmesh-controller)

*Testing:*
- Tested installation and end to end controller with both rc5 and rc-preview tags

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: stable/appmesh-controller/Chart.yaml [hunk 1]
```diff
@@ -1,19 +1,16 @@
 apiVersion: v1
 name: appmesh-controller
 description: App Mesh controller Helm chart for Kubernetes
-version: 0.6.1
-appVersion: 0.5.0
+version: 0.6.2
```
Comment (M00nF1sh) [inline stable/appmesh-controller/Chart.yaml:4]: let's bump this chart version to a new major version like v1.0.0 as well

File: stable/appmesh-controller/README.md [hunk 2]
```diff
@@ -4,6 +4,20 @@ App Mesh controller Helm chart for Kubernetes
...
 
+**Note** App Mesh controller is a release candidate. Please use it for testing purpose only as it has backward incompatible changes with v0.5.0. Upgrade instructions will be provided before the final release.
+
+**Note** If you wish to use App Mesh preview features, you can add `-preview` to the published image. e.g. `v1.0.0-rc4-preview`
+Two important things when using App Mesh preview:
+1. When configuring IAM policies, use `appmesh-preview` as the service name instead of `appmesh`. See the example JSON below.
+2. When configuring pods, add the following annotation so Envoy sidecars point to the preview as well:
+```
+annotations:
+  appmesh.k8s.aws/preview: "true"
```
Comment (M00nF1sh) [inline stable/appmesh-controller/README.md:15]: i believe we changed from true to enabled

Comment (fawadkhaliq) [reply stable/appmesh-controller/README.md:15]: Yes, thank you for catching this! It was marked to update and missed

File: stable/appmesh-controller/ci/values.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
-# CI testing values for appmesh-controller.
+# CI testing values for appmesh-controller
 
 region: us-west-2
 image:
-  repository: stefanprodan/aws-app-mesh-controller-for-k8s
-  tag: v0.2.0
+  repository: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon/appmesh-controller
+  tag: v1.0.0-rc5
```
Comment (bcelenza) [inline stable/appmesh-controller/ci/values.yaml:6]: Shouldn't the default be the current stable version (0.2.0) with an option to opt-in to the release candidate?

Comment (bcelenza) [reply stable/appmesh-controller/ci/values.yaml:6]: Also, image repository should be regionalized.

Comment (fawadkhaliq) [reply stable/appmesh-controller/ci/values.yaml:6]: The charts will be not published by this check-in so current users will continue to get v0.2.0. Charts will be published when we build a new tag at the release time. Keeping v0.2.0 here will not help as all the configuration changes/crds are not compatible with v0.2.0 image

Comment (fawadkhaliq) [reply stable/appmesh-controller/ci/values.yaml:6]: > Also, image repository should be regionalized.

That's a good idea to make it configurable. I will mark this for a follow up change

**Pull request merged**